bot/context.js

```diff
@@ -1,14 +1,29 @@
 'use strict';
 
 const warn = require('../actions/warn');
+const ban = require('../actions/ban');
+const { scheduleDeletion } = require('../utils/tg');
 
-const config = require('../config');
+const {
+	warnInlineKeyboard,
+	deleteWarnsAfter = false,
+	deleteBansAfter = false,
+} = require('../config');
 
-const reply_markup = { inline_keyboard: config.warnInlineKeyboard };
+const normalisedDeleteWarnsAfter = typeof deleteWarnsAfter === 'object'
+	? deleteWarnsAfter : { auto: deleteWarnsAfter, manual: deleteWarnsAfter };
```

Comment:
```suggestion
	? deleteWarnsAfter
	: { auto: deleteWarnsAfter, manual: deleteWarnsAfter };
```

I basically don't like keeping ternary operator in 2 lines. Either 1 or 3.
I'll try adding [`"multiline-ternary":`](https://eslint.org/docs/rules/multiline-ternary)`["error", "always-multiline"]` to `.eslintrc`.

Reply:
Resolved in  27710a0